app/router.js

```diff
@@ -6,16 +6,19 @@ import { get } from '@ember/object';
 import { scheduleOnce } from '@ember/runloop';
 
 const Router = EmberRouter.extend({
-  navbar: service(),
   location: config.locationType,
   rootURL: config.rootURL,
+  navbar: service(),
   metrics: service(),
   fastboot: service(),
```

Comment:
```suggestion
  fastboot: service(),
  router: service(),
```

Reply:
I've tried this and I get an error:

```
Uncaught RangeError: Maximum call stack size exceeded at http://localhost:7357/assets/vendor.js, line 23173
```

Reply:
Tweaked the suggestion...

Reply:
I can't the invocation right, and I don't know what I'm doing wrong 🤔

Reply:
@locks would upgrading beyond the Ember 3.5.x that we’re on here help?